python/cudf_polars/cudf_polars/dsl/expressions/datetime.py

```diff
@@ -135,6 +134,30 @@ def do_evaluate(
             for child in self.children
         ]
         (column,) = columns
+        if self.name is TemporalFunction.Name.Week:
+            result = plc.strings.convert.convert_integers.to_integers(
+                plc.strings.convert.convert_datetime.from_timestamps(
+                    column.obj,
+                    format="%V",
+                    input_strings_names=plc.interop.from_arrow(
```

Comment:
Surely it must be possible to convert from a datetime to the week without going through a string

Reply:
Unfortunately theres no `WEEK` [datetime component](https://docs.rapids.ai/api/libcudf/stable/group__datetime__extract#ga4b452b569dd909cda36718348dc43fb5), which would be the easiest thing. FWIW, this is also what the [cudf.pandas implementation](https://github.com/rapidsai/cudf/blob/79109d435bf2b1221d8eef88b9b66e3bd8083166/python/cudf/cudf/core/column/datetime.py#L535-L541) does. However, like in https://github.com/rapidsai/cudf/pull/18171, there are now two separate consumers that could benefit from the presence of a libcudf implementation here, which would be the best solution IMO.